index.bs

```diff
@@ -5943,6 +6012,11 @@ Each stored [=virtual authenticator=] has the following properties:
     [=User Verification=] will always succeed. If set to [FALSE], it will fail.
 
     Note: This property has no effect if |hasUserVerification| is set to [FALSE].
+: |extensions|
+:: A string array containing the [=extension identifiers=] supported by the [=Virtual Authenticator=].
+
+[=Virtual authenticators=] MUST support all [=authenticator extensions=] present in its |extensions| array.
+They MUST NOT support any [=authenticator extension=] not present in its |extensions| array.
```

Comment:
If the virtual authnr does not support any extensions, is this property's value an empty array, or is the property absent? Ah, I see below in "Add Virtual Authenticator" that the default is an empty array.